joss_paper_0.18/paper.md

```diff
@@ -0,0 +1,39 @@
+---
+title: 'SciPy 0.18: Open source toolbox for science and engineering'
+tags:
+  - science
+  - engineering
+  - mathematics
+authors:
+ - name: Evgeni Burovski
+   orcid: 0000-0001-8149-0483
+   affiliation: National Research University Higher School of Economics
+date: XXX
+bibliography: paper.bib
+---
+
+# Summary
+
+SciPy (pronounced "Sigh Pie") is open-source software for mathematics, science,
+and engineering. It includes modules for statistics, optimization, integration,
+linear algebra, Fourier transforms, signal and image processing, ODE solvers,
+and more. 
+
+The @SciPy library depends on @NumPy, which provides convenient and fast N-dimensional
+array manipulation. The SciPy library is built to work with NumPy arrays, and
+provides many user-friendly and efficient numerical routines such as routines
+for numerical integration and optimization. 
+
+The latest stable release of SciPy is [0.18.1](@SciPy0181), which improves on the feature
+release [0.18.0](SciPy0180). Together they contain numerous bug-fixes, improved test coverage,
+better documentation and many new features. For the complete description of the
```

Comment:
documentation and many -> documentation, and many

to be consistent with the other Oxford commas